app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="crayons-card p-5">
+  <% if page.image %>
+    <div class="s:max-w-50 mb-4">
+      <img src="<%= page.image %>" height="auto" alt="Avatar">
+    </div>
+  <% end %>
+  <div class="color-primary">
```

Comment:
Is this class needed here? I believe it _should_ inherit color from the article.